bitmap-destruction-force/model/grayscale-constructor.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.grayScale = function(buf, err){
+  if (err) throw new Error('Error grayscaling bitmap');
+  var duck = new exports.GrayScale(buf);
+  duck.grayScale(buf);
+};
+
+exports.GrayScale = function(buf){
+  this.filesize = buf.readUInt32LE(2);
+  this.offset = buf.readUInt32LE(10);
+  this.colorArray = buf.slice(54, 1078);
+};
+
+exports.GrayScale.prototype.grayScale = function(){
+  var currentColor;
+  for (var i = 0;i < 1024;i += 4){
```

Comment:
Instead of hard-coding in 1024, it would be a good idea to reference a property such as this.colorArray.length.